js/src/layers/Path.js

```diff
@@ -27,6 +27,19 @@ export class LeafletPathModel extends vectorlayer.LeafletVectorLayerModel {
 export class LeafletPathView extends vectorlayer.LeafletVectorLayerView {
   model_events() {
     super.model_events();
+    var key;
+    var o = this.model.get('options');
+    for (var i = 0; i < o.length; i++) {
+      key = o[i];
+      this.listenTo(
+        this.model,
+        'change:' + key,
+        function() {
+          this.obj.setStyle(this.get_options());
+        },
+        this
+      );
+    }
     this.obj.setStyle(this.get_options());
```

Comment:
This line is unexpected :open_mouth: It should not be needed

Reply:
Let's remove it later

Reply:
:+1: